source/administration/changelog.md

```diff
@@ -4,6 +4,120 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.2
+
+### Security Update
+
+- Mattermost v5.2.0 contains medium level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Highlights
+
+#### Embed Mattermost in Other Apps (Beta)
+ - Added support for extensions, which allow you to embed Mattermost in other apps and websites via OAuth 2.0.
+ - A sample extension for Chrome [is here](https://github.com/mattermost/mattermost-chrome-extension). For documentation on how to write an extension, see here. // XXX Add a link to doc when ready
+
+#### Plugins
+ - Added support to add/delete and enable/disable plugins via the CLI.
+ - See our [demo plugin](https://github.com/mattermost/mattermost-plugin-demo) that demonstrates the capabilities of a Mattermost plugin. For a starting point to write a Mattermost plugin, see our [sample plugin](https://github.com/mattermost/mattermost-plugin-sample).
+ - Breaking changes to the plugins framework introduced. To migrate your existing plugins to be compatible with Mattermost 5.2 and later, see our [migration guide](https://developers.mattermost.com/extend/plugins/migration/).
+ 
+#### Searching Archived Channels
+ - Added ability to search for archived channels on desktop and mobile clients.
```

Comment:
Added ability to search for archived channel content on desktop and mobile clients.